demisto_sdk/__main__.py

```diff
@@ -3155,19 +3155,32 @@ def error_code(ctx, config, **kwargs):
     help="Whether or not to include dependencies.",
     default=False,
 )
+@click.option(
+    "-npv",
+    "--no-python-version",
```

Comment:
please change it to be `--python-version/--no-python-version`, default=`False`, for readability.